apps/shooting-clubs/controllers/confirm.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Controller = require('../../common/controllers/confirm');
+
+module.exports = class ConfirmController extends Controller {
+
+  getFieldData(field, req) {
+    const defaults = super.getFieldData(field, req);
+
+    if (field.field === 'location-addresses') {
+      return defaults.value.map(address => {
+        address['address-category'] = [].concat(address['address-category']);
```

Comment:
just because if `super.getFieldData` returned a reference to an item, we would actually be changing this value in the loop. Totes agree that this should be field level formatting